tests/callbacks/test_gpu_stats_monitor.py

```diff
@@ -63,6 +63,37 @@ def test_gpu_stats_monitor(tmpdir):
         assert any(f in h for h in met_data.dtype.names)
 
 
+@RunIf(min_gpus=1)
+def test_gpu_stats_monitor_no_queries(tmpdir):
+    """
+    Test GPU stats are logged using a logger.
+    """
+    model = BoringModel()
+    gpu_stats = GPUStatsMonitor(
+        memory_utilization=False,
+        gpu_utilization=False,
+        intra_step_time=True,
+        inter_step_time=True,
+        fan_speed=False,
+        temperature=False,
+    )
+    logger = CSVLogger(tmpdir)
+    log_every_n_steps = 2
+
+    trainer = Trainer(
+        default_root_dir=tmpdir,
+        max_epochs=2,
+        limit_train_batches=7,
+        log_every_n_steps=log_every_n_steps,
```

Comment:
Running 2 batches can be enough here

```suggestion
        fast_dev_run=2
```

Reply:
done